src/serialize.rs

```diff
@@ -231,6 +231,171 @@ pub fn serialized_length_from_bytes(b: &[u8]) -> io::Result<u64> {
     Ok(f.position())
 }
 
+use crate::sha2::{Digest, Sha256};
```

Comment:
Does `clippy` complain about this not being at the top? Seems like it should.

Reply:
it doesnt